pallets/polkadex-ido/src/lib.rs

```diff
@@ -101,11 +101,11 @@ pub mod weights;
 use pallet_polkadex_ido_primitives::{FundingRoundWithPrimitives, StringAssetId, VoteStat};
 pub use weights::WeightInfo;
 
-#[cfg(test)]
-mod mock;
-
-#[cfg(test)]
-mod test;
+// #[cfg(test)]
+// mod mock;
+//
+// #[cfg(test)]
+// mod test;
```

Comment:
why commenting this?

Reply:
We are going to revamp IDO Pallet completely. So there is no need to fix tests of the old IDO Pallet. As we are going to remove it.